swarm-event-schema.json

```diff
@@ -272,6 +272,9 @@
         "numberPlate": {
            "type": "string"
         },
+        "numberPlateImage": {
+           "type": "string"
```

Comment:
For non-standard / structured strings there should be further format information attached. 
- i.e. "data-time" for string
- or something like
` {
       "type": "string",
       "contentEncoding": "base64",
       "contentMediaType": "image/jpeg" <-?
   }`
see  https://datatracker.ietf.org/doc/html/draft-bhutton-json-schema-validation-00#section-8.6 for image data.